presto-main/src/main/java/io/prestosql/security/AccessControlConfig.java

```diff
@@ -13,23 +13,32 @@
  */
 package io.prestosql.security;
 
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableList;
 import io.airlift.configuration.Config;
 
+import javax.validation.constraints.NotNull;
+
 import java.io.File;
+import java.util.List;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
 
 public class AccessControlConfig
 {
-    private File accessControlFile;
+    private List<File> accessControlFiles = ImmutableList.of();
```

Comment:
I'd prefer to have single file which can list sequence of access controls -- https://github.com/prestosql/presto/issues/1852

i can draft a proposal myself on top of your change and we can see how it goes. 
cc @electrum @martint because this is a broader topic